lib/OpenQA/BuildResults.pm

```diff
@@ -131,13 +131,24 @@ sub compute_build_results ($group, $limit, $time_limit_days, $tags, $subgroup_fi
         return \%result;
     }
 
+    # build sorting
+    # 1: sort by build name (default)
+    # 0: sort by newest job in build
+    # 2: sort by oldest job in build
```

Comment:
These numbers should be constants and with those constants you can remove this comment.